### DISCUSSION TITLE
Discussion about jwt token usage?

### DISCUSSION THREAD

**Discussion by hurovroman** [hurovroman/eCommerce-NextJS-Sanity#2] [Announcements]
Now, I am modifying the ecommerce website usin jwt token for login logic.
But I am not sure how to store it on frontend side. Please let me know how to do it?

**Comment by mythofmidas** [comment]
Hello, Roman. I have some ways to sore jwt token on frontend side.

1: Local Storage
localStorage.setItem('token', 'your-jwt-token');

2: Session Storage
sessionStorage.setItem('token', 'your-jwt-token');

These are main usages to do it. Hope you are doing well.

**Comment by hurovroman** [comment]
And then another ways?

**Comment by mythofmidas** [comment]
Of course:
HTTP-Only Cookies:

More secure as they are not accessible via JavaScript, reducing the risk of XSS attacks.
Slightly more complex to implement and handle on the server side.

Here's code
import axios from 'axios';

axios.post('your-login-endpoint', { username, password })
  .then(response => {
    document.cookie = `token=${response.data.token}; HttpOnly`;
  });

**Comment by hurovroman** [comment]
Hmm, Thanks a lot.
